resources/views/mail/community/annual-recap.blade.php

```diff
@@ -0,0 +1,115 @@
+}
+
+$completionClause = '';
+$numCompletions = $countThing($recapData['numCompletions'], 'game');
+$numBeaten = $countThing($recapData['numBeaten'], 'game');
+if ($numCompletions) {
+    if ($numBeaten) {
+        $completionClause = "You completed $numCompletions, and beat an additional $numBeaten on softcore.";
+    } else {
+        $completionClause = "You completed $numCompletions.";
+    }
+} elseif ($numBeaten) {
+    $completionClause = "You beat $numBeaten on softcore.";
+}
+
+@endphp
+
+<x-mail::message>
+Congratulations {{ $user->display_name }}!
+
+In {{ $recapData['year'] }}, you played {{ $countThing($recapData['gamesPlayed'], 'game') }} on <a href="{{ route('home') }}">retroachievements.org</a> and unlocked {{ $countThing($recapData['achievementsUnlocked'], 'achievement') }}, earning you {{ $hardcorePointsClause }}{{ $softcorePointsClause }}. {{ $leaderboardsSubmittedClause }}
+
+You spent {{ $recapData['totalPlaytime'] }} playing games {{ $playedSystemsClause }}.
+
+{{ $masteryClause }}{{ $completionClause }}
+
+@if ($recapData['mostPlayedGame'])
+<x-mail::image-panel
+    src="{{ media_asset($recapData['mostPlayedGame']->ImageIcon) }}"
+    alt="{{ $recapData['mostPlayedGame']->Title }} game badge"
+    url="{{ route('game.show', $recapData['mostPlayedGame']) }}"
+    width="64"
+    height="64"
+>
+    Your most played game was <a href="{{ route('game.show', $recapData['mostPlayedGame']) }}">{{ $recapData['mostPlayedGame']->Title }}</a> at {{ $recapData['mostPlayedGamePlaytime'] }}.
+</x-mail::panel>
+@endif
+
+@if ($recapData['rarestHardcoreAchievement'])
+<x-mail::image-panel
+    src="{{ $recapData['rarestHardcoreAchievement']->badge_unlocked_url }}"
+    alt="{{ $recapData['rarestHardcoreAchievement']->Title }} achievement badge"
+    url="{{ route('achievement.show', $recapData['rarestHardcoreAchievement']) }}"
+    width="64"
+    height="64"
+>
+    Your rarest achievement earned was <a href="{{ route('achievement.show', $recapData['rarestHardcoreAchievement']) }}">{{ $recapData['rarestHardcoreAchievement']->Title }}</a> from {{ $recapData['rarestHardcoreAchievement']->Game->Title }}, which has only been earned in hardcore by {{ $recapData['rarestHardcoreAchievementEarnRate'] }}% of players.
+</x-mail::image-panel>
+@elseif ($recapData['rarestSoftcoreAchievement'])
+<x-mail::image-panel
+    src="{{ $recapData['rarestSoftcoreAchievement']->badge_unlocked_url }}"
+    alt="{{ $recapData['rarestSoftcoreAchievement']->Title }} achievement badge"
+    url="{{ route('achievement.show', $recapData['rarestSoftcoreAchievement']) }}"
+    width="64"
+    height="64"
+>
+    Your rarest achievement earned was <a href="{{ route('achievement.show', $recapData['rarestSoftcoreAchievement']) }}">{{ $recapData['rarestHardcoreAchievement']->Title }}</a> from {{ $recapData['rarestHardcoreAchievement']->Game->Title }}, which has only been earned by {{ $recapData['rarestSoftcoreAchievementEarnRate'] }}% of players.
```

Comment:
Possible copy+paste error: There are references on this line to the rarest hardcore achievement.